eng/dependabot/Packages.props

```diff
@@ -3,6 +3,8 @@
   <!-- Packages in this file have versions updated periodically by Dependabot. Versions managed by Darc/Maestro should be in ..\Packages.props. -->
 
   <ItemGroup>
+    <PackageVersion Include="WixToolset.Dtf.WindowsInstaller" Version="5.0.2" />
```

Comment:
WiX 5 cannot be used right now. There are internal requirements that need to be met and that work won't be completed until next year